docs/config.json

```diff
@@ -454,6 +454,10 @@
               "title": "SQL Server (Preview)",
               "slug": "/database-access/guides/sql-server-ad/"
             },
+            {
+              "title": "Self-Hosted Redis",
```

Comment:
Nit: Maybe move this above SQL Server section? To keep it grouped with other self-hosted guides and also move the preview SQL Server to the bottom.